PostGreKeyRotation/PostGreKeyRotation/Models/SimpleUser.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace PostGreKeyRotation.Models
+{
+    public class SimpleUser
```

Comment:
Not to nitpick but perhaps this file should be my commit?

Reply:
Fixed